frontend/views/containers/group-settings/GroupLeaveModal.vue

```diff
@@ -93,7 +99,22 @@ export default {
       this.$refs.modal.close()
     },
     async submit () {
-      alert('TODO implement this')
+      if (this.$v.form.$invalid) {
+        return
+      }
+
+      try {
+        const message = await sbp('gi.contracts/group/removeOurselves/create',
+          {
+            groupId: this.currentGroupId
+          },
+          this.currentGroupId
+        )
+        await sbp('backend/publishLogEntry', message)
```

Comment:
All this type of stuff should no longer be done in Vue files. Instead it should be moved into `frontend/controller/actions/group.js`.

Feel free to boyscout other vue.js files and move any leftover `/create` messages into the actions folder too (there are a few I saw that are still in vue files).

Reply:
changed to actions. Did the same to Save Group Settings.